ts/features/fims/history/store/selectors/index.ts

```diff
@@ -0,0 +1,18 @@
+import * as O from "fp-ts/Option";
+import { pipe } from "fp-ts/lib/function";
+import { backendStatusSelector } from "../../../../../store/reducers/backendStatus";
+import { GlobalState } from "../../../../../store/reducers/types";
+
+export const fimsHistoryPotSelector = (state: GlobalState) =>
+  state.features.fims.history.consentsList;
+
+// the flag should be treated as enabled when either true or undefined,
+// and is defined as an optional bool
+export const fimsIsHistoryEnabledSelector = (state: GlobalState) =>
+  pipe(state, backendStatusSelector, status =>
+    pipe(
```

Comment:
No need for a second pipe, just process the output from `backendStatusSelector` with the `O.map(_ => _.config ...)`

PS: what do you think of renaming `_` to `backendStatus`, in order to ease the reading?

Reply:
right, that flew over my head 😅 
fixed it nonetheless, LMK!